.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,20 @@
+name: golangci-lint
+on:
+  pull_request:
+permissions:
+  contents: read
+jobs:
+  golangci:
+    name: golangci-lint-net-rpc
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.28.3
+          working-directory: ./net/rpc
+          args: --issues-exit-code=0
```

Comment:
Instead of running it from `net/rpc` directory, maybe we should use a `.golangci.yml` to exclude issues in other paths?  

Doesn't `args: --issues-exit-code=0` mean it will never fail CI? I think the purpose of adding this it to fail the CI check, so I think we'd want to remove that line.

Reply:
Thanks for the call outs here! Turned on `verbose`.

> maybe we should use a `.golangci.yml` to exclude issues in other paths?

hm, i may be wrong here but since this is a "monorepo" I think we need to specify the cwd.